corehq/apps/geospatial/templates/gps_capture_view.html

```diff
@@ -24,7 +24,9 @@
 {% registerurl 'paginate_mobile_workers' domain %}
 {% initial_page_data 'case_types_with_gps' case_types_with_gps %}
 {% initial_page_data 'couch_user_username' couch_user_username %}
+{% registerurl 'geo_polygons' domain %}
```

Comment:
Curious of the reason behind adding this ?

Reply:
This was added as the GPS Capture page was throwing a JS exception because it was missing a reference to this URL. This is because the `geo_polygons` URL is fetched as a global variable in the `models.py` file. 

Looking at it again however, it's only being referenced in a single location, so it might be better to have the variable as a local reference in the function where its needed and then this `registerurl` can be removed.

Reply:
I converted both global URL references to local ones for the `models.py` file ([edb3a61](https://github.com/dimagi/commcare-hq/pull/35126/commits/edb3a61959713458392466733bb86670ef0836ed))